### PR TITLE
RS-206 - Elasticsearch example

### PR DESCRIPTION
**Pull request by lagache**

### CODE REVIEW

File: examples/elasticsearch/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "reshuffle-elastic-search-example",
+  "version": "1.0.0",
+  "description": "Examples for Google Sheets Connector",
```
Comment (duckranger) [inline examples/elasticsearch/package.json:4]: Inconsistency detected during operation `=`

File: examples/elasticsearch/package.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "main": "GoogleSheetsExamples.js",
```
Comment (duckranger) [inline examples/elasticsearch/package.json:5]: This won't work :(

File: examples/elasticsearch/package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  "scripts": {
+    "sheets": "node elastic.js",
+    "sheetsAutoNLP": "node elastic.js"
+  },
```
Comment (duckranger) [inline examples/elasticsearch/package.json:9]: ?

**Review by duckranger — CHANGES_REQUESTED** [review]

File: examples/elasticsearch/package.json (hunk 1 above)
Comment (lagache) [reply examples/elasticsearch/package.json:4]: hahaha

File: examples/elasticsearch/package.json (hunk 3 above)
Comment (lagache) [reply examples/elasticsearch/package.json:9]: yeah... I rushed the example obviously

**Review by duckranger — APPROVED** [review]

**Pull request merged**